src/CodeTemplates.js

```diff
@@ -0,0 +1,19 @@
+export const ReactTemplates = {
+  'ButtonHero': ({label, icon, dense, type, state}) => {
```

Comment:
can this go into the ButtonComponent as well?

Reply:
Yes, I planned to export this from the ButtonCatalog file and import it here so I can export this map. Make sense?